packages/storybook8/eslint.config.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
```

Comment:
With this many rules and exceptions, I wonder if the Storybook eslint config should be integrated with the rest of the mono-repo config or left as an independent.

Reply:
perhaps independant definitely worth a discussion

Reply:
reverted this change as per our discussion.